include/tiled.h

```diff
@@ -54,18 +69,21 @@ Tiled *tiled_init(MemZone *memory_pool, sprite_t *sprite, const char *map_path,
  * @param screen_rect
  *        Rect of the current screen. Used to cull tiles outside of the screen.
  */
```

Comment:
I believe you forgot to add `view_position` to the doc here.

Reply:
Good catch